python/langsmith/client.py

```diff
@@ -814,6 +827,9 @@ def request_with_retries(
                         f"Connection error caused failure to {method} {pathname}"
                         f"  in LangSmith API. {recommendation}."
                         f" {repr(e)}"
+                        f"\nContent-Length: {content_length}"
+                        f"\nAPI Key: {masked_api_key}"
```

Comment:
Great idea.  I assume this will look something like what we show in the LS UI?

Reply:
Example from me:

```
WARNING:langsmith.client:Failed to batch ingest runs: langsmith.utils.LangSmithConnectionError: Connection error caused failure to POST https://api.smith.langchain.com/runs/batch  in LangSmith API. Please confirm your internet connection.. SSLError(MaxRetryError("HTTPSConnectionPool(host='api.smith.langchain.com', port=443): Max retries exceeded with url: /runs/batch (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2393)')))"))
Content-Length: 105000932
API Key: lsv2*********************************************33
post: trace=fec2a5d9-bd40-4238-a7f0-ed4f23096d36,id=fec2a5d9-bd40-4238-a7f0-ed4f23096d36
```